analyzer/codechecker_analyzer/analyzers/clangsa/ctu_manager.py

```diff
@@ -134,20 +138,33 @@ def generate_ast(triple_arch, action, source, config, env):
                   cmdstr, err)
 
 
-def ast_dump_path(source_path):
+def ast_dump_path(path, config, triple_arch):
     """ AST-dump based analysis uses preprocessed paths, here the path prefix
     'ast' and the filename suffix '.ast' is added to the name of the original
     source file. """
 
+    # If clang-extdef-mapping can read a pch then the file entry is an absolute
+    # path to the ast file. Let's transform that to a relative path from the
```

Comment:
This is only true, and only needed, for Clang PCHs, right? (I am concerned about accidentally running into GCC PCHs.)

Reply:
Right.